test/python/qasm/example_version_2.qasm

```diff
@@ -0,0 +1,11 @@
+OPENQASM 2;
```

Comment:
The fact that `OPENQASM 2;` works now is kind of a side-effect - it'd be best for the good test file to use `2.0`.

Reply:
For this one we have a few tests that run with 2.0 and I wanted to keep at least one that shows that indeed a single-digit version will work

Reply:
That's a good idea, let's leave it as it is.